### PR TITLE
feat: add `see more/see less` for product properties

### PR DESCRIPTION
**Pull request by vkrashenko**
Description:
https://virtocommerce.atlassian.net/browse/ST-3891

QA-test:

Demo-test:

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-1.19.0-pr-430-d15c-d15c973a.zip

### CODE REVIEW

File: client-app/shared/catalog/components/product/properties.vue [hunk 1]
```diff
@@ -6,10 +6,17 @@
...
   >
     <!-- Properties -->
-    <ProductProperty v-for="property in groupedProperties" :key="property.name" :label="property.name" class="mb-4">
+    <ProductProperty v-for="property in propertiesToShow" :key="property.name" :label="property.name" class="mb-4">
       {{ property.values }}
     </ProductProperty>
-
+    <a
+      v-if="product.properties && product.properties.length >= MAX_DISPLAY_ITEMS"
+      class="my-0.5 !h-10 text-blue-600 dark:text-blue-500 underline hover:cursor-pointer decoration-dashed"
```
Comment (goldenmaya) [inline client-app/shared/catalog/components/product/properties.vue:14]: 1. use --color-link and --color-link-hover variables
2. do not use !h-10

File: client-app/shared/catalog/components/product/properties.vue [hunk 2]
```diff
@@ -6,10 +6,17 @@
     :title="model.title || $t('shared.catalog.product_details.technical_specs_block_title')"
   >
     <!-- Properties -->
-    <ProductProperty v-for="property in groupedProperties" :key="property.name" :label="property.name" class="mb-4">
+    <ProductProperty v-for="property in propertiesToShow" :key="property.name" :label="property.name" class="mb-4">
       {{ property.values }}
     </ProductProperty>
-
+    <a
```
Comment (goldenmaya) [inline client-app/shared/catalog/components/product/properties.vue:12]: I'm not sure but I think it should be placed after Vendor property.

File: client-app/shared/catalog/components/product/properties.vue [hunk 3]
```diff
@@ -6,10 +6,17 @@
...
+      v-if="product.properties && product.properties.length >= MAX_DISPLAY_ITEMS"
+      class="my-0.5 !h-10 text-blue-600 dark:text-blue-500 underline hover:cursor-pointer decoration-dashed"
+      @click="showAll = !showAll"
+    >
+      <i class="md:hidden px-2 fas fa-save text-2xl" />
```
Comment (goldenmaya) [inline client-app/shared/catalog/components/product/properties.vue:17]: use `<VcIcon name="chevron-down" size="xs" />`

File: client-app/shared/catalog/components/product/properties.vue [hunk 4]
```diff
@@ -18,15 +18,29 @@
     >
       <Vendor :vendor="product.vendor" withRating />
     </ProductProperty>
+
+    <a
+      v-if="product.properties && product.properties.length > MAX_DISPLAY_ITEMS"
+      class="flex gap-x-1 mb-4 items-center text-14 text-[color:var(--color-link)] hover:text-[color:var(--color-link-hover)] hover:cursor-pointer underline decoration-dashed"
+      @click="showAll = !showAll"
+    >
+      <VcIcon class="text-[color:var(--color-primary)]" :name="[showAll ? 'chevron-up' : 'chevron-down']" size="xs" />
```
Comment (muller39) [inline client-app/shared/catalog/components/product/properties.vue:27]: chamge 'atributes' order
class="text-[color:var(--color-primary)]" :name="[showAll ? 'chevron-up' : 'chevron-down']" size="xs"
to
:name="[showAll ? 'chevron-up' : 'chevron-down']" size="xs" class="text-[color:var(--color-primary)]"

File: client-app/shared/catalog/components/product/properties.vue [hunk 5]
```diff
@@ -18,15 +18,29 @@
...
+      {{
+        showAll == true ? $t("shared.catalog.product_details.see_less") : $t("shared.catalog.product_details.see_more")
```
Comment (muller39) [inline client-app/shared/catalog/components/product/properties.vue:29]: 'showAll' has boolean type, it coud be simply?
showAll ? $t("shared.catalog.product_details.see_less") : $t("shared.catalog.product_details.see_more")

File: client-app/shared/catalog/components/product/properties.vue [hunk 6]
```diff
@@ -18,15 +18,29 @@
...
+      }}
+    </a>
   </ProductTitledBlock>
 </template>
 
 <script setup lang="ts">
-import { computed, PropType } from "vue";
+import { computed, PropType, ref } from "vue";
 import _ from "lodash";
 import { Product } from "@/xapi/types";
```
Comment (muller39) [inline client-app/shared/catalog/components/product/properties.vue:36]: xapi inclide export * from "./types";

import { Product } from "@/xapi/types";
to
import { Product } from "@/xapi";

File: client-app/shared/catalog/components/product/properties.vue [hunk 7]
```diff
@@ -18,15 +18,29 @@
...
 import { prepareProperties, ProductProperty, ProductTitledBlock, Vendor } from "@/shared/catalog";
 
+const MAX_DISPLAY_ITEMS = 8;
+const showAll = ref(false);
+
 const props = defineProps({
```
Comment (muller39) [inline client-app/shared/catalog/components/product/properties.vue:42]: mb change 'const props = defineProps(...' to ower interface solution? [like here](https://github.com/VirtoCommerce/vc-theme-b2b-vue/pull/432/files#diff-bfd252f92c636f553a414b5b0033155be478d69dae522045afad8719879f23e5:~:text=%7D)%3B-,interface%20Props%20%7B,-cart%3A%20CartType)

**Review by muller39 — APPROVED** [review]

File: client-app/shared/catalog/components/product/properties.vue [hunk 8]
```diff
@@ -18,26 +18,36 @@
     >
       <Vendor :vendor="product.vendor" withRating />
     </ProductProperty>
+
+    <a
+      v-if="product.properties && product.properties.length > MAX_DISPLAY_ITEMS"
```
Comment (artmakarov) [inline client-app/shared/catalog/components/product/properties.vue:23]: use `groupedProperties.length > MAX_DISPLAY_ITEMS`

File: client-app/shared/catalog/components/product/properties.vue [hunk 9]
```diff
@@ -18,26 +18,36 @@
...
+      class="flex gap-x-1 mb-4 items-center text-14 text-[color:var(--color-link)] hover:text-[color:var(--color-link-hover)] hover:cursor-pointer underline decoration-dashed"
+      @click="showAll = !showAll"
+    >
+      <VcIcon :name="[showAll ? 'chevron-up' : 'chevron-down']" size="xs" class="text-[color:var(--color-primary)]" />
```
Comment (artmakarov) [inline client-app/shared/catalog/components/product/properties.vue:27]: `:name="showAll ? 'chevron-up' : 'chevron-down'`

**Review by artmakarov — CHANGES_REQUESTED** [review]

File: locales/en.json [hunk 10]
```diff
@@ -454,7 +454,9 @@
           "variation_sku_label": "@:common.labels.sku",
           "price_label": "@:common.labels.product_price",
           "vendor_label": "@:shared.catalog.product_details.vendor_label"
-        }
+        },
+        "see_more": "See more",
+        "see_less": "See less"
```
Comment (artmakarov) [inline locales/en.json:459]: move to `common.buttons` section

**Review by artmakarov — APPROVED** [review]

**Pull request merged**